playground/infrastructure/helper.py

```diff
@@ -212,31 +214,36 @@ def _validate(tag: dict, supported_categories: List[str]) -> bool:
     """
     valid = True
     for field in fields(TagFields):
-        if tag.get(field.default) is None:
+        if field.default not in tag:
             logging.error("tag doesn't contain " + field.default + " field: " + tag.__str__() + "\n" +
                           "Please, double-check that this field exists in the beam playground tag. " +
                           "If you are sure that this field exists in the tag double-check the format of indenting.")
-            valid = False
+            return False
+
+    name = tag.get(TagFields.NAME)
+    if name == '':
+        logging.error("tag's field name is incorrect: " + tag.__str__() + "\n" +
+                      "name can not be empty.")
+        return False
```

Comment:
It is better to use `valid = False` and after all checks return the result of validation. In that case, user will see all his typos with tag.

Reply:
Agree, it was decided during Aydar's work on this validation.

Reply:
Done